process_report/tests/unit/processors/test_coldfront_fetch_processor.py

```diff
@@ -95,3 +100,31 @@ def test_coldfront_project_not_found(self, mock_get_allocation_data):
             str(cm.exception),
             f"Projects {answer_project_set} not found in Coldfront and are billable! Please check the project names",
         )
+
+    @mock.patch(
+        "process_report.processors.coldfront_fetch_processor.ColdfrontFetchProcessor._fetch_coldfront_allocation_api",
+    )
+    def test_nonbillable_clusters(self, mock_get_allocation_data):
+        """What happens when an invoice project is not found in Coldfront."""
```

Comment:
So how is this testing nonbillable clusters? it just seems to be testing projects that don't have information in coldfront. Would it have mattered if in your test invoice the cluster name wasn't `ocp-test`?

Reply:
The Coldfront processor throws an error if an invoice project is 1. Not found in Coldfront, and 2. Is billable.
This test checks that even though Coldfront has no info on Projects `P3` and `P4`, it did not throw an error because these projects belong to the nonbillable `ocp-test` cluster.

If `P3` and `P4` did not belong to `ocp-test`, an error would be raised

Reply:
Thanks for the clarification. Can you update the docstring to be more explicit then? Maybe something like:

"""
No errors are raised when an invoice project belonging to a non billable cluster (ocp-test) is not found in Coldfront
"""

> If P3 and P4 did not belong to ocp-test, an error would be raised

Ah, and we are testing that in `test_coldfront_project_not_found`.

Reply:
The docstring has been amended